src/EnergyPlus/Construction.cc

```diff
@@ -1919,6 +1920,20 @@ void ConstructionProps::reportTransferFunction(EnergyPlusData &state, int const
     }
 }
 
+void ConstructionProps::reportLayers(EnergyPlusData &state)
+{
+    // Report the layers for each opaque construction in predefined tabular report
+    // J. Glazer March 2024
+    auto &opd = state.dataOutRptPredefined;
```

Comment:
I don't think you ever used this shortcut you made.  I'd say either remove it or use it in the loop.  In this tiny little function with just a couple uses, it's fine either way.  Also maybe I just can't see the usage here on GH.

Reply:
Thanks, I removed that line.